packages/components/src/public/files/WebDav.ts

```diff
@@ -1,8 +1,8 @@
 import { Map, Record } from 'immutable';
 
 import { ActionURL, Ajax, Utils } from '@labkey/api';
+import {DEFAULT_FILE, IFile} from "../..";
 
-import { DEFAULT_FILE, IFile } from './models';
 
 export class WebDavFile extends Record(DEFAULT_FILE) implements IFile {
```

Comment:
Just curious, are you using all of the exported items from this file in the client module? I.e. the `WebDavFile`, `getWebDavFiles`, and `uploadWebDavFile`?

Reply:
Nevermind, I found my answer from the other PR (and the answer is Yes).